content/docs/reference/config/installation-options-version-comparison/index.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Installation Options - Generate differences between current release and previous release (v1.0)
```

Comment:
Can you shorten the title?  Something like "Installation options added and removed in 1.1"

Reply:
@sdake I have given it a try. Will let you verify.  Thanks.

Reply:
I think this document should be hidden and not even have a title in the index.
There should be a link to it from the `Installation Options` page, right at the top. Something like:
```
See [installation option release changes ](<link>) for the differences between the current release and the previous one.
```

@geeknoid @rcaballeromx WDYT?

Reply:
Why hide it? If it's important enough to be on the web site, then shouldn't be listed in the normal index?

The site doesn't currently have support for content that doesn't show up in the various indices. I'd have to add some annotations to induce this behavior.

Reply:
Hidden or not, I think the most important thing is that there is a link to it from the `Installation Options` page, which IMO is the reference document that people would go to for options info.

The release-specific changes, if not hidden, isn't really a reference doc to me, that's why I think it seems out of place. Maybe there's a good place for something like this somewhere closer to the release notes?

Reply:
@frankbu where we put it I leave up to you and @geeknoid.  I do believe this is helpful for operators to get a feel for what has changed.  I'm not sure this PR will merge prior to friday, but I'd like this work to backport to 1.1 when it is ready.

Cheers
-steve